src/scheduler.c

```diff
@@ -544,10 +543,12 @@ JL_DLLEXPORT jl_task_t *jl_task_get_next(jl_value_t *trypoptask, jl_value_t *q,
                 // so make sure io_loop_tid is notified to check wait_empty
```

Comment:
@vtjnash could you update this comment to reflect what you think is the expected behaviour here?